Oubliette_Password_Manager/README.md

```diff
@@ -16,11 +16,15 @@ Hash creation example:
       $oubliette-idea$9c846ab8c1dc703330521e7ca77489beded5d23b3aa821edea8f0324fbdb9f08
     oubliette-idea-complex-withdata-v15.oub
       $oubliette-idea$9c846ab8c1dc703330521e7ca77489beded5d23b3aa821edea8f0324fbdb9f08
-
-Testing the complex password requires target-enc set to iso-8859-1:
-
+```
+Testing the passwords:
+```
     cat oubliette-passwords.txt >> run/password.lst
-    john -target-enc=iso-8859-1 oubliette-blowfish-complex.hash
-    john -target-enc=iso-8859-1 oubliette-blowfish-complex-withdata-v15.hash
-    john -target-enc=iso-8859-1 oubliette-idea-complex.hash
-    john -target-enc=iso-8859-1 oubliette-idea-complex-withdata-v15.hash
+    john oubliette-blowfish-complex.hash
+    john oubliette-blowfish-complex-withdata-v15.hash
+    john oubliette-idea-complex.hash
+    john oubliette-idea-complex-withdata-v15.hash
```

Comment:
I'd use simply `john -w=oubliette-passwords.txt *.hash` instead of these many commands.